db/seeds.rb

```diff
@@ -5,3 +5,15 @@
 #
 #   movies = Movie.create([{ name: "Star Wars" }, { name: "Lord of the Rings" }])
 #   Character.create(name: "Luke", movie: movies.first)
+
+if Doorkeeper::Application.count.zero?
+  Doorkeeper::Application.create!(name: "Web Client", redirect_uri: "", scopes: "")
+  Doorkeeper::Application.create!(name: "iOS Client", redirect_uri: "", scopes: "")
+  Doorkeeper::Application.create!(name: "Android Client", redirect_uri: "", scopes: "")
```

Comment:
```suggestion
  Doorkeeper::Application.create!(name: 'Web Client', redirect_uri: '', scopes: '')
  Doorkeeper::Application.create!(name: 'iOS Client', redirect_uri: '', scopes: '')
  Doorkeeper::Application.create!(name: 'Android Client', redirect_uri: '', scopes: '')
```

Let's use `single quote` whenever possible 🙌